libraries/ts-command-line/src/providers/CommandLineParameterProvider.ts

```diff
@@ -881,19 +881,26 @@ export abstract class CommandLineParameterProvider {
     );
     if (required && environmentVariable) {
       // Add some special-cased logic to handle required parameters with environment variables
+
+      const originalPreParse: (() => void) | undefined = parameter._preParse?.bind(parameter);
       parameter._preParse = () => {
         // Set the value as non-required before parsing. We'll validate it explicitly
         argparseArgument.required = false;
+        originalPreParse?.();
```

Comment:
Wondering if this should go before our tweaking? The latter one happens after we revert our tweaking, so to the implementor, nothing has changed